BlueprintUI/Sources/Layout/Column.swift

```diff
@@ -2,10 +2,46 @@ import UIKit
 
 /// Displays a list of items in a linear vertical layout.
 public struct Column: StackElement {
+    /// Describes how the column's children will be horizontally aligned.
+    public enum ColumnAlignment: Equatable {
+        /// Children will be stretched to fit the horizontal size of the column.
+        case fill
+        /// Using the specified alignment, children will be aligned relatively to each other, and
+        /// then all the contents will be aligned to the column's bounding box.
```

Comment:
I would mention the fact that you can using `leading` etc. without having to use this directly

Reply:
Sure, I'll update this doc to make it clearer that it's only needed for custom alignments.